### PR TITLE
{math}[foss/2015a] Octave 4.0.0 (REVIEW)

### PR DESCRIPTION
**Pull request by tvandera**
Bump Octave to v 4.0.0

### CODE REVIEW

File: easybuild/easyconfigs/o/Octave/Octave-4.0.0-foss-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+homepage = 'http://www.gnu.org/software/octave/'
+description = """GNU Octave is a high-level interpreted language, primarily intended for numerical computations."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+    ('PCRE', '8.36'),
```
Comment (boegel) [inline easybuild/easyconfigs/o/Octave/Octave-4.0.0-foss-2015a.eb]: bump to 8.37?

File: easybuild/easyconfigs/o/Octave/Octave-4.0.0-foss-2015a.eb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    ('ncurses', '5.9'),
```
Comment (boegel) [inline easybuild/easyconfigs/o/Octave/Octave-4.0.0-foss-2015a.eb]: 6.0?

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/o/Octave/Octave-4.0.0-foss-2015a.eb (hunk 1 above)
Comment (tvandera) [inline easybuild/easyconfigs/o/Octave/Octave-4.0.0-foss-2015a.eb]: I can do this. Would you prefer?

File: easybuild/easyconfigs/o/Octave/Octave-4.0.0-foss-2015a.eb (hunk 2 above)
Comment (tvandera) [inline easybuild/easyconfigs/o/Octave/Octave-4.0.0-foss-2015a.eb]: I can do this. Would you prefer?

File: easybuild/easyconfigs/o/Octave/Octave-4.0.0-foss-2015a.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/o/Octave/Octave-4.0.0-foss-2015a.eb]: we try to update versions of dependencies when we have an easyconfig for a new software version, so yes please

File: easybuild/easyconfigs/o/Octave/Octave-4.0.0-foss-2015a.eb (hunk 2 above)
Comment (boegel) [inline easybuild/easyconfigs/o/Octave/Octave-4.0.0-foss-2015a.eb]: yes please :)

File: easybuild/easyconfigs/o/Octave/Octave-4.0.0-foss-2015a.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/o/Octave/Octave-4.0.0-foss-2015a.eb]: with yes please I mean: yes, please update your PR by bumping this version yourself

File: easybuild/easyconfigs/o/Octave/Octave-4.0.0-foss-2015a.eb (hunk 2 above)
Comment (wpoely86) [inline easybuild/easyconfigs/o/Octave/Octave-4.0.0-foss-2015a.eb]: no, keep it to 5.9. We're gonna have version trouble else. 
For the 2016a toolchains, we will use version 6.0

Comment (boegel) [inline easybuild/easyconfigs/o/Octave/Octave-4.0.0-foss-2015a.eb]: Yeah, +1, I forgot about this. Keep ncurses to 5.9 when using the `2015a` or `2015b` toolchains, for compatibility with other modules where ncurses is a dependency (e.g. Python) (but do bump PCRE).

**Comment by boegel** [comment]
Looks good, will get a test report in here, but should be good to go, thanks @tvandera!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/ca3e54fcaf4315c7aa2e for a full test report.

**Pull request merged**